packages/cms-cli/commands/lint.js

```diff
@@ -27,51 +27,75 @@ const {
 } = require('../lib/usageTracking');
 
 const COMMAND_NAME = 'lint';
+const DESCRIPTION = 'Lint a file or folder for HubL syntax';
 
-function configureCommand(command) {
-  command
-    .version(version)
-    .description('lint a file or folder for HubL syntax')
-    .arguments('<path>')
-    .action(async filepath => {
-      setLogLevel(command);
-      logDebugInfo(command);
-      const { config: configPath } = command;
-      loadConfig(configPath);
-      checkAndWarnGitInclusion();
+const parseOptions = async options => {
```

Comment:
Since there are side effects, I am not sure `parseOptions` is the best name here.

Reply:
Changed